lib/Launcher.js

```diff
@@ -45,6 +45,7 @@ const DEFAULT_ARGS = [
   '--disable-prompt-on-repost',
   '--disable-sync',
   '--disable-translate',
+  '--font-render-hinting=medium',
```

Comment:
If this is really better, we should probably set it as the default in headless chromium and not pass it in with Puppeteer. @caseq ptal.